frontend/src/metabase/admin/settings/utils.spec.js

```diff
@@ -0,0 +1,20 @@
+import { prepareAnalyticsValue } from './utils'
+
+describe('prepareAnalyticsValue', () => {
+    const defaultSetting = { value: 120, type: 'number' }
+
+    const checkResult = (setting = defaultSetting, expected = "success") =>
```

Comment:
@tlrobinson Yeah. Sometimes all the per test boilerplate gets annoying to me, but I can see this being overkill.